CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 0.73.0 / 2024-04-03
+
+* [CHANGE] Add proxyURL validation for servicemon,podmonitor and probe #6464
+      The fix might be a breaking change for those who rely on the config reloader to expand env variables in the config.
```

Comment:
```suggestion
     *It could be a breaking change for those who rely on the configuration reloader sidecar to expand environment variables in the generated configuration file.*
```